diff -Naur iptables-1.4.17.old/extensions/GNUmakefile.in iptables-1.4.17/extensions/GNUmakefile.in
--- iptables-1.4.17.old/extensions/GNUmakefile.in	2012-12-25 12:38:36.000000000 +0000
+++ iptables-1.4.17/extensions/GNUmakefile.in	2012-12-30 02:45:53.195775525 +0000
@@ -101,7 +101,8 @@
 	ln -fs $< $@
 
 # Need the LIBADDs in iptables/Makefile.am too for libxtables_la_LIBADD
-ip6t_NETMAP_LIBADD  = -lip6tc
+ip6t_NETMAP_LIBADD  = -L${top_builddir}/libiptc/.libs \
+                      -L${top_builddir}/libiptc -lip6tc
 xt_RATEEST_LIBADD   = -lm
 xt_statistic_LIBADD = -lm
 
